packages/liquidator/src/liquidationStrategy.js

```diff
@@ -0,0 +1,223 @@
+const assert = require("assert");
+// This module is responsible for sizing liquidation positions.
+// If a defensive strategy is enabled, it will check withdraw positions
+// to see if they require an extension rather than full liquidation.
+module.exports = (config, libs, emit = x => x) => {
+  const {
+    // how much money to set aside for defense in tokens
+    whaleDefenseFundWei = 0,
+    // when to activate defense, when withdraw is X% in time complete
+    defenseActivationPercent,
```

Comment:
nit: do you not want to set a default value for `defenseActivationPercent`

Reply:
its not used at all unless whaleDefense is activated. If activated an error will be thrown if a value is not provided.